packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -19,7 +19,7 @@ import { BaseWidget } from '@theia/core/lib/browser';
 import { CommandLineOptions } from '@theia/process/lib/common/shell-command-builder';
 import { TerminalSearchWidget } from '../search/terminal-search-widget';
 import { TerminalProcessInfo } from '../../common/base-terminal-protocol';
-
+import URI from '@theia/core/lib/common/uri';
```

Comment:
💄 Please keep the empty line between the imports and types. It helps the code readability. Thank you!

Reply:
Sure, thanks!